sickbeard/tv.py

```diff
@@ -1480,6 +1480,9 @@ def downloadSubtitles(self, force=False):
 
             subliminal.save_subtitles(foundSubs, directory=subs_new_path, single=not sickbeard.SUBTITLES_MULTI)
 
+            if not sickbeard.EMBEDDED_SUBTITLES_ALL and sickbeard.SUBTITLES_EXTRA_SCRIPTS and self.location.endswith('mkv'):
```

Comment:
It should be working/complete, but I want the original requester to try it first.